### PR TITLE
Add enhancement into lisav2

### PR DESCRIPTION
**Pull request by lubaihua33**
1. Add judgement for VMData if null then throw
2.  Add retries when calling Get-AzNetworkInterface API
3. Change udp test ntttcp version to master

### CODE REVIEW

File: Testscripts/Linux/utils.sh [hunk 1]
```diff
@@ -2745,6 +2745,8 @@ function install_lagscope () {
 
 # Build and install ntttcp
 function build_ntttcp () {
+	# Clean up ntttcp before install
+	rm -rf /usr/local/bin/ntttcp
```
Comment (squirrelsc) [inline Testscripts/Linux/utils.sh]: Can we update ntttcp to use same version for udp and tcp?

File: Libraries/Azure.psm1 [hunk 2]
```diff
@@ -2495,6 +2495,19 @@ Function Set-SRIOVinAzureVMs {
 			$AllNics = Get-AzNetworkInterface -ResourceGroupName $ResourceGroup `
 			| Where-Object { $($_.VirtualMachine.Id | Split-Path -leaf) -eq $VMName }
 
+			# Sometimes Get-AzNetworkInterface returns null, retry 5 times
+			$retry_count = 0
+			While ($null -eq $AllNics -and $retry_count -lt 5) {
```
Comment (squirrelsc) [inline Libraries/Azure.psm1]: try more times, like 120. It means two minutes.

File: Libraries/Azure.psm1 [hunk 3]
```diff
@@ -2495,6 +2495,19 @@ Function Set-SRIOVinAzureVMs {
...
+				$AllNics = Get-AzNetworkInterface -ResourceGroupName $ResourceGroup `
+				| Where-Object { $($_.VirtualMachine.Id | Split-Path -leaf) -eq $VMName }
+				$retry_count++
+				Start-Sleep -Seconds 1
```
Comment (squirrelsc) [inline Libraries/Azure.psm1]: check if AllNics has value, don't sleep.

File: Testscripts/Linux/utils.sh (hunk 1 above)
Comment (lubaihua33) [reply Testscripts/Linux/utils.sh]: Yes, updated in the xml file.

File: Libraries/Azure.psm1 (hunk 2 above)
Comment (lubaihua33) [reply Libraries/Azure.psm1]: Done

File: Libraries/Azure.psm1 (hunk 3 above)
Comment (lubaihua33) [reply Libraries/Azure.psm1]: Done

File: Testscripts/Linux/utils.sh (hunk 1 above)
Comment (squirrelsc) [reply Testscripts/Linux/utils.sh]: Does the ntttcp of master branch support both udp and tcp now? I mean to update ntttcp, not in lisa.

Comment (lubaihua33) [reply Testscripts/Linux/utils.sh]: Yes, master branch can support udp and tcp.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**